### ISSUE TITLE
Multiple Docker for Mac host nodes in Swarm Mode all have same "Docker Desktop" hostname and it's unclear how to change them

### ISSUE THREAD

**Issue by bionicles** [docker/for-mac#4080]
### Expected behavior
When you join multiple macs to a docker swarm, it should be clear how to change the hostname right from within the app (without needing to do a bunch of googling)

### Actual behavior
Multiple Docker for Mac hosts on a docker swarm don't "just work" because the host name is always "Docker Desktop" -- you can't easily tell which is which and scripts/features break

### Information
  - Is it reproducible? Sure, start a swarm and join two macs to it. They'll both have the same hostname. This breaks certain swarm tasks (like looping over nodes to get ipv4)
  - Is the problem new? No
  - Did the problem appear with an update? No
  - macOS Version: Catalina (latest)
  - Docker for Mac: version... Stable 2.1.0.5
  - Diagnostics: 544B1AC7-0E55-4A06-9473-2468C51FD0F9/20191128180146

### Steps to reproduce the behavior
1. start a swarm
2. join two macs to the swarm
3. on a manager type "docker node ls" and you'll see the macs are all named "Docker Desktop"
4. try to figure out how to change the hostname
5. it is not clear how to change that from docs and app ui

TLDR: Please add a "hostname" input field to the docker for mac app so users in swarm mode can easily change the hostname. Or if this is done with daemon.json please document it!
